pkg/rule/pull_request_label_rule.go

```diff
@@ -0,0 +1,25 @@
+package rule
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openshift/patchmanager/pkg/config"
+	"github.com/openshift/patchmanager/pkg/github"
+)
+
+type PullRequestLabelRule struct {
+	Config *config.PullRequestLabelRuleConfig
+}
+
+func (p *PullRequestLabelRule) Evaluate(pullRequest *github.PullRequest) ([]string, bool) {
+	result := []string{}
+	for _, l := range pullRequest.Issue.Labels {
+		for _, c := range p.Config.RefuseOnLabel {
+			if strings.HasPrefix(l.GetName(), c) {
```

Comment:
if we're going to treat the provided value as a prefix, should we just go all the way to treating it as a regex for matching purposes?

Reply:
only if we never make David patch manager :-)